src/main/java/com/exactpro/th2/common/schema/util/Log4jConfigUtils.kt

```diff
@@ -0,0 +1,61 @@
+package com.exactpro.th2.common.schema.util
+
+import org.apache.log4j.PropertyConfigurator
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.slf4j.LoggerFactory
+import java.net.MalformedURLException
+import java.nio.file.Files
+import java.nio.file.Path
+
+class Log4jConfigUtils {
+
+    private val LOGGER = LoggerFactory.getLogger(Log4jConfigUtils::class.java)
+
+    fun configure(
+        pathList: List<String>,
+        firstVersionFileName: String,
+        secondVersionFileName: String,
+        secondVersionPriority: Boolean = true
+    ) {
+        val configPathList: Pair<Path, Int>? = pathList
```

Comment:
Maybe convert to sequence to avoid packing to a collection at the end of each operation.